annotation/consume_trigger.go

```diff
@@ -130,6 +140,13 @@ func (t *TriggerIfDeepNonNil) equals(other ConsumingAnnotationTrigger) bool {
 	return false
 }
 
+// Copy returns a deep copy of this ConsumingAnnotationTrigger
+func (t *TriggerIfDeepNonNil) Copy() ConsumingAnnotationTrigger {
+	copyConsumer := *t
+	t.Ann = t.Ann.copy()
```

Comment:
Same as the above here :)

Reply:
(same as above. Fixed and added unit test to safeguard :) )